### PR TITLE
feat: UX: Multichain: Release the send page

### PR DESCRIPTION
**Pull request by darkwing**
## **Description**

Removes the feature flag surrounding the new send flow, as well as removes legacy send files.

## **Related issues**

N/A

## **Manual testing steps**

1. Go to Send page
2. See the new UI
3. Complete a send
4. Go to an NFT page
5. Click send
6. Send the NFT appropriately

## **Pre-merge author checklist**

- [ ] I’ve followed [MetaMask Coding Standards](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/CODING_GUIDELINES.md).
- [ ] I've clearly explained what problem this PR is solving and how it is solved.
- [ ] I've linked related issues
- [ ] I've included manual testing steps
- [ ] I've included screenshots/recordings if applicable
- [ ] I’ve included tests if applicable
- [ ] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [ ] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.
- [ ] I’ve properly set the pull request status:
  - [ ] In case it's not yet "ready for review", I've set it to "draft".
  - [ ] In case it's "ready for review", I've changed it from "draft" to "non-draft".

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

**Comment by darkwing** [comment]
Blocked on: https://consensyssoftware.atlassian.net/browse/MMASSETS-131

**Comment by darkwing** [comment]
Presently blocked by: https://github.com/MetaMask/metamask-extension/pull/22551

**Comment by hesterbruikman** [comment]
@darkwing Does it make sense to you if I add a `DO NOT MERGE` label and add the Send flow bug fixes epic [#2106](https://github.com/MetaMask/MetaMask-planning/issues/2106) as a blocker?

File: app/_locales/pl/messages.json [hunk 1]
```diff
@@ -127,6 +127,9 @@
   "cancel": {
     "message": "Anuluj"
   },
+  "cancelEdit": {
+    "message": "Cancelar edição"
```
Comment (cryptotavares) [inline app/_locales/pl/messages.json:131]: This is a portuguese translation... pretty sure it shouldn't be on the polish translations file

File: test/e2e/helpers.js [hunk 2]
```diff
@@ -760,36 +760,9 @@ const sendScreenToConfirmScreen = async (
...
-        (_found) => {
-          driver.clickElement({
-            text: 'I understand',
-            tag: 'button',
-          });
-        },
-        (error) => {
-          console.error('Element not found.', error);
-        },
-      );
```
Comment (cryptotavares) [inline test/e2e/helpers.js:780]: I don't think that this should be removed. The toolTipApproveButton label is used in the [SendPageContent component](https://github.com/MetaMask/metamask-extension/blob/develop/ui/components/multichain/pages/send/components/content.tsx#L83-L103), that is used on the new send page.

File: test/e2e/helpers.js [hunk 3]
```diff
@@ -760,36 +760,9 @@ const sendScreenToConfirmScreen = async (
...
-    });
-  } else {
-    await driver.clickElement({
-      text: 'Next',
-      tag: 'button',
-      css: '[data-testid="page-container-footer-next"]',
-    });
-  }
+  await driver.clickElement({ text: 'Continue', tag: 'button' });
+  await driver.clickElement({ text: 'Confirm', tag: 'button' });
```
Comment (cryptotavares) [inline test/e2e/helpers.js:765]: Remove clicking on the confirm button. We do not want to submit the transaction. The `sendScreenToConfirmScreen` helper is used for ppom tests, and on those tests we use mainnet (for ppom support.. we mock all network requests)

File: ui/pages/routes/routes.component.js [hunk 4]
```diff
@@ -371,11 +370,7 @@ export default class Routes extends Component {
           path={`${CONFIRM_TRANSACTION_ROUTE}/:id?`}
           component={ConfirmTransaction}
         />
-        <Authenticated
-          path={SEND_ROUTE}
-          component={process.env.MULTICHAIN ? SendPage : SendTransactionScreen}
-          exact
-        />
+        <Authenticated path={SEND_ROUTE} component={SendPage} exact />
```
Comment (cryptotavares) [inline ui/pages/routes/routes.component.js:373]: With this it seems that we are no longer using the SendTransactionScreen page anywhere. Are we going to remove all the dead code that will linger around after this PR?

**Review by digiwand — COMMENTED** [review]
code lgtm! +1 on @cryptotavares's comments

**Review by bowensanders — DISMISSED** [review]
[Snaps] LGTM :) ✅

**Review by BZahory — DISMISSED** [review]

**Review by sleepytanya — DISMISSED** [review]

**Review by BZahory — APPROVED** [review]

**Review by sleepytanya — APPROVED** [review]

**Review by bowensanders — APPROVED** [review]
LGTM

**Pull request merged**